server/server.go

```diff
@@ -1,10 +1,17 @@
 package server
 
 import (
+	"context"
+	"encoding/json"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"strconv"
+	"time"
+
+	"github.com/joshprzybyszewski/cribbage/server/middleware"
+
+	"github.com/joshprzybyszewski/cribbage/auth"
```

Comment:
🌵 import order -- these go with other in-repo imports

Reply:
👍